frontend/packages/dev-console/src/components/import/advanced/ServerlessScalingSection.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-unused-vars, no-undef */
```

Comment:
This shouldn't be necessary. Also we should not disable any eslint rules.

Reply:
Fixed